webhooks/testdata/new_dunning_event_notification.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let's get tests in here for both before and after credit invoices are turned on. It is required that everyone turn it on by November 1, 2018, and when that happens invoices will no longer have a `SubscriptionUUID` field. There's an example of what the new response will look like [here](https://dev.recurly.com/page/webhooks#dunning-event-notifications).

Reply:
Please put this comment above the test for the post-credit-invoices webhook: 

```
// Available after credit invoices have been turned on in your account. See https://docs.recurly.com/docs/credit-invoices-release#section-updates-to-the-api-and-sdks for more information. 
```

That way after the November 1 deadline has passed it will be easier to remove deprecated code.